Source/CBLMisc.m

```diff
@@ -299,6 +299,9 @@ BOOL CBLIsOfflineError( NSError* error ) {
     if ($equal(domain, NSURLErrorDomain))
         return code == NSURLErrorDNSLookupFailed
             || code == NSURLErrorNotConnectedToInternet
+            || code == NSURLErrorCannotConnectToHost
+            || code == NSURLErrorCannotFindHost
+            || code == NSURLErrorNetworkConnectionLost
```

Comment:
These error codes don't mean the device is offline, just that the server isn't reachable. I don't think it would be a good idea to have the replicator go offline if, say, the server URL has the wrong port number; this should be reported as an error.